environments/includes/selenium.base.yml

```diff
@@ -1,27 +1,8 @@
 version: "3.5"
 services:
-  selenium-hub:
-    container_name: ${WARDEN_ENV_NAME}_selenium-hub
-    hostname: selenium-hub
-    image: selenium/hub:latest
-    healthcheck:
-      test: /opt/bin/check-grid.sh --host 0.0.0.0 --port 4444
-      interval: 20s
-      retries: 3
-    environment:
-      GRID_TIMEOUT: 0
-      GRID_BROWSER_TIMEOUT: 0
-
   selenium-chrome:
     hostname: ${WARDEN_ENV_NAME}_selenium-chrome
-    image: selenium/node-chrome${WARDEN_SELENIUM_DEBUG:-}:3.8.1
+    image: selenium/standalone-chrome${WARDEN_SELENIUM_DEBUG:-}:latest
```

Comment:
I suggest to not use `latest`.
It's better to have predictability of what is going to be installed.
I suggest to define a variable, like on the other images.

Reply:
```suggestion
    image: selenium/standalone-chrome${WARDEN_SELENIUM_DEBUG:-}:${WARDEN_SELENIUM_VERSION:-3.141.59}
```